src/Configuration/Dotenv.hs

```diff
@@ -1,86 +1,66 @@
 -- |
 -- Module      :  Configuration.Dotenv
--- Copyright   :  © 2015–2016 Stack Builders Inc.
+-- Copyright   :  © 2015–2017 Stack Builders Inc.
 -- License     :  MIT
 --
 -- Maintainer  :  Stack Builders <hackage@stackbuilders.com>
 -- Stability   :  experimental
 -- Portability :  portable
 --
--- This module contains common functions to load and read dotenv files.
+-- Provides the ability to load a dotenv file with an specific configuration.
 
-{-# LANGUAGE CPP #-}
+{-# LANGUAGE CPP             #-}
+{-# LANGUAGE RecordWildCards #-}
 
 module Configuration.Dotenv
-  ( load
```

Comment:
@CristhianMotoche it seems we're still breaking backward compatibility here without providing a replacement. Why are we doing this? Are we sure no one is using this functionality?